apps/apps.h

```diff
@@ -233,6 +234,8 @@ int set_cert_times(X509 *x, const char *startdate, const char *enddate,
             "Disallow initial connection to servers that don't support RI"}, \
         {"allow_no_dhe_kex", OPT_S_ALLOW_NO_DHE_KEX, '-', \
             "In TLSv1.3 allow non-(ec)dhe based key exchange on resumption"}, \
+        {"prioritize_chacha", OPT_S_PRIORITIZE_CHACHA, '-', \
+            "Prioritize ChaCha ciphers for mobile clients"}, \
```

Comment:
The wording here should also change to drop "mobile" and just say "when preferred by client" or some such

Reply:
Also perhaps shorten "prioritize" to "prefer"

Reply:
"Prefer ChaCha ciphers when preferred by client" sounds awkward to me...